server/api/api.go

```diff
@@ -124,3 +128,74 @@ func (h *Handler) setLink(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(status)
 	_, _ = w.Write([]byte(`{"status": "OK"}`))
 }
+
+func (h *Handler) deleteLink(w http.ResponseWriter, r *http.Request) {
+	autolinkName := r.URL.Query().Get(autolinkclient.AutolinkNameQueryParam)
+	if autolinkName == "" {
+		h.handleError(w, errors.New("autolink name or pattern should not be empty"))
+		return
+	}
+
+	links := h.store.GetLinks()
+	found := false
+	for i := 0; i < len(links); i++ {
```

Comment:
```suggestion
	for i, link := range links {
```
I thing we can do this. this is more clear

Reply:
this was not working here as in this I am removing an element so we have to calculate the length because range doesn't get updated